OSTI Energy Citations.js

```diff
@@ -200,4 +200,4 @@ var testCases = [
 		]
 	}
 ]
-/** END TEST CASES **/
\ No newline at end of file
+/** END TEST CASES **/
```

Comment:
The first test case is not working for me in Scaffold (actually it is running but never ending). But it works when directly typed the url in firefox and tested it.